ui/app/css/itcss/components/item.scss

```diff
@@ -0,0 +1,116 @@
+.Item__flex-grid {
```

Comment:
We usually place scss files meant for usage with one component in the same directory as the components js files. I'd recommend that here, unless you had a reason not to.